docs/sources/flowspec.md

```diff
@@ -49,7 +49,7 @@ CLI syntax to add ipv4/ipv6 flowspec rule is
                     { port | destination-port | source-port | icmp-type | icmp-code | packet-length | dscp | label } <ITEM>... }...
    <PROTO> : ospf, pim, igp, udp, igmp, tcp, egp, rsvp, gre, ipip, unknown, icmp, sctp, <VALUE>
    <FRAGMENT_TYPE> : dont-fragment, is-fragment, first-fragment, last-fragment, not-a-fragment
-   <TCPFLAG> : rst, push, ack, urgent, fin, syn
+   <TCPFLAG> : U, C, E, F, S, R, P, A
```

Comment:
Please update [ `tcp-flags [not] [match] <TCPFLAG> ...`](https://github.com/osrg/gobgp/pull/1228/files#diff-e647e3b1d45f1144a2b744811179ce30L48) too.

Also, [here](https://github.com/osrg/gobgp/blob/master/gobgp/cmd/global.go#L866) needs the same update.